codegen.mts

```diff
@@ -50,7 +49,7 @@ const config: CodegenConfig = {
           resolversNonOptionalTypename: {
             interfaceImplementingType: true,
             unionMember: true,
-            excludeTypes: ['TokenInfoPayload'],
+            excludeTypes: ['TokenInfoPayload', 'Schema', 'GraphQLNamedType'],
```

Comment:
These abstract types are added here as we don't use `__typename` to handle them.